container-onnxruntime-bundle/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<!-- Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                          http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>container-onnxruntime-bundle</artifactId>
```

Comment:
Consider removing redundant `-bundle` suffix. Modules with prefix `container-` are all bundles.

Reply:
Agree, consistent naming is always good.